### ISSUE TITLE
Comment indentation in switch statement

### ISSUE THREAD

**Issue by rastersize** [nicklockwood/SwiftFormat#181]
Hey there!

Just integrated SwiftFormat (0.29.8) and found something that seems be be a bug. The following code:
```swift
switch (location) {
// Have a sip of whiskey.
case .bar:
    takeSip(of: 🥃)
}
```

Is indented as:
```swift
switch (location) {
    // Have a sip of whiskey.
case .bar:
    takeSip(of: 🥃)
}
```

Which seems off to me. I’d expect it to not change at all. Cheers!

**Comment by nicklockwood** [comment]
You’re right, that does seem off. I’ll look into it - thanks!

**Comment by rastersize** [comment]
Thanks! :)

**Comment by nicklockwood** [comment]
Fixed in 0.29.9

**Comment by rastersize** [comment]
Thanks for fixing so quickly!

**Comment by nicklockwood** [comment]
@tengyifei thanks for reporting this. It looks like I didn't test it with the `--comments ignore` option.

**Comment by nicklockwood** [comment]
@tengyifei this was fixed in 0.35.10 but I forgot to close the issue. Sorry about that!
